Cartfile.resolved

```diff
@@ -1 +1 @@
-github "ReactiveX/RxSwift" "4.2.0"
+github "ReactiveX/RxSwift" "4.3.1"
```

Comment:
Any reason for bumping minimum version here?

Reply:
My bad, I just did a carthage update when bootstrap would have been enough :)